products/satellite.md

```diff
@@ -41,11 +41,7 @@ versionCommand: yum info satellite
 
 ---
 
-[Red Hat® Satellite](https://www.redhat.com/technologies/management/satellite) is an infrastructure management product specifically designed to keep 
-Red Hat Enterprise Linux® environments and other Red Hat infrastructure running efficiently, with security, and compliant with various standards.
-
-There is a new minor release approximately every 6 months.
-
-[Lifecycle Table](https://access.redhat.com/support/policy/updates/satellite)
+"[Red Hat Satellite](https://www.redhat.com/technologies/management/satellite) is an infrastructure management product specifically designed to keep 
+Red Hat Enterprise Linux® environments and other Red Hat infrastructure running efficiently, with security, and compliant with various standards."
```

Comment:
```suggestion
Red Hat Enterprise Linux environments and other Red Hat infrastructure running efficiently, with security, and compliant with various standards."
```

Would like to follow Wikipedia here and say no to Trademark/Copyright/Registered Trademark symbols: https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Trademarks#General_rules